MainModule/Client/Core/Anti.lua

```diff
@@ -621,13 +619,18 @@ return function(Vargs, GetEnv)
 				end
 
 				--// Check Context Level
-				local ran, _ = pcall(function()
-					local test = Instance.new("StringValue")
+				local test
+				local ran = pcall(function()
+					test = Instance.new("StringValue")
 					test.RobloxLocked = true
 				end)
 				if ran then
 					Detected("crash", "RobloxLocked usable")
 				end
+				if test then
```

Comment:
you could make this `elseif test then` to make the code slightly shorter

Reply:
Should be fixed now.